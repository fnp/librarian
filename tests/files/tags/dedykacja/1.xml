<autor_utworu>Władysław Stanisław Reymont</autor_utworu>
<dzielo_nadrzedne>Chłopi</dzielo_nadrzedne>
<nazwa_utworu>Część pierwsza --- Jesień</nazwa_utworu>

<dedykacja>
<strofa>
Miriamowi/
(Zenonowi Przesmyckiemu)
</strofa>
</dedykacja>
