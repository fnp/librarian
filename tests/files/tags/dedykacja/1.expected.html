<div id="book-text">
<h1>
<span class="wl author" id="i1">Władysław Stanisław Reymont</span><span class="wl collection" id="i2">Chłopi</span><span class="wl title" id="i3">Część pierwsza — Jesień</span>
</h1>
<div class="dedication" id="i5">
<div class="stanza" id="i4">
<a href="#f1" class="wl-num">1</a><div class="wl verse" id="f1">Miriamowi</div>
<div class="wl verse" id="f2">(Zenonowi Przesmyckiemu)</div>
</div>
</div>
</div>
